packages/design-system/src/components/OcTable/OcTable.vue

```diff
@@ -497,6 +497,7 @@ export default defineComponent({
   }
 
   &-hover tr:hover td:not(:last-child) span:not(.avatarInitials):not(button span) {
+    // Not excluding .avatarInitials -> ?
     color: var(--oc-color-swatch-brand-hover) !important;
```

Comment:
Has no effect on `avatarInitials` at a first glance, no? I remember that the avatars in the user- and group-lists had this ugly hover effect, but it seems to be gone with your `color: white !important;` on the `avatarInitials` class.

Reply:
`color: white !important` is a workaround. `:not(.avatarInitials)` should work but it doesn't 🤷

Reply:
Ahh okay, got it. You need to specifically de-select all `span`'s. Otherwise, the color will just "move" to the next one. 

`&-hover tr:hover td:not(:last-child) span:not(.oc-avatar):not(.avatarInitials):not(button span)`

Reply:
removed the exception rule and rely on `color: white !Important` on .avatarInitials to make the behaviour globally.